lib/Support/LoweringOptions.cpp

```diff
@@ -172,6 +174,8 @@ std::string LoweringOptions::toString() const {
     options += "emitBindComments,";
   if (omitVersionComment)
     options += "omitVersionComment,";
+  if (noVerilogLocation)
+    options += "noVerilogLocation,";
```

Comment:
Please add docs for all new lowering options.

Also add it to  `LoweringOptionsParser.h` .